src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -260,6 +260,9 @@ export class SentryAppExternalIssueForm extends React.Component<Props, State> {
       required,
     };
 
+    //async only used for select components
+    const isAsync = typeof field.async === 'undefined' ? true : field.async; //default to true
```

Comment:
@scefali should we do `!!field.async`, incase it's null?

Reply:
@NisanthanNanthakumar `null` is not a valid choice (we have validation to prevent this). Also, the point of this is logic that we want `undefined` to evaluate to `true` so using `!!field.async` doesn't work (the default value is `true`).

Reply:
@scefali ahh cool. yeah I meant to use it at the end of the ternary operator. 
`typeof field.async === 'undefined' ? true : !!field.async;`
As long as there exists some validation, it's unnecessary.